### PR TITLE
Editing guide in Manual and Charge Types page

### LINKED ISSUE #1384: Diverted DUII "More Info" entry says "not added yet"
The rules are defined on the Class, so investigate this bug.

### PR DESCRIPTION
**Pull request by wittejm**
Resolves #1384, #1377 , #1308 
Charge Types are given in a separate page that is linked in the Editing Results section in the Manual, and in the Edit Charge Panel. This means a user will only come across it after they start digging into Editing. 
   
I finally abandoned the idea of a Complete Document of Time Eligibility. The list of charge types is still essential for use of the Editing feature though. 

This PR changes the `expungement_rules`of many charge types. They're no longer being formatted with markdown and, because I wanted to retain the bullet-pointed lists, I chose to define the content from the backend with some nested tuples that get used to build HTML in the frontend. 

There are two small spacing issues I couldn't figure out. @hmarcks Could you tackle these? 

Overhanging icon:
overhang_editing_icon

Off-center:
off-center-type-icon

### CODE REVIEW

File: src/frontend/src/components/Manual/index.tsx [hunk 1]
```diff
@@ -539,20 +547,19 @@ class Manual extends React.Component {
...
+                be directed to input identifying information. Inputting
+                identifying information is optional but can help save you time.
               </p>
               <p className="mb4">
-                After you input the information, RecordSponge will generate 
-                a .zip file with PDFs of the expungement paperwork for all 
-                of the charges.
+                After you input the information, RecordSponge will generate a
+                .zip file with PDFs of the expungement paperwork for all of the
+                charges.
```
Comment (KentShikama) [inline src/frontend/src/components/Manual/index.tsx:562]: One PDF per case with any eligible charge

File: src/frontend/src/components/Manual/index.tsx [hunk 2]
```diff
@@ -598,64 +605,66 @@ class Manual extends React.Component {
                     </p>
                     <h4 className="fw7 mb2">Conviction versus Arrest</h4>
                     <p className="mb3">
-                      If the Disposition of a case is “Dismissed,” use an Arrest Form.
+                      If the Disposition of a case is “Dismissed,” use an Arrest
```
Comment (KentShikama) [inline src/frontend/src/components/Manual/index.tsx:608]: If there are no convicted charges in the case

File: src/frontend/src/components/Rules/index.tsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  };
+
+  fetchRules = () => {
+    axios
+      .request({
+        url: "/api/rules",
+        method: "get",
+      })
+      .catch((error) => {
+        return Promise.reject(error);
```
Comment (KentShikama) [inline src/frontend/src/components/Rules/index.tsx:35]: Is this necessary?

Comment (wittejm) [reply src/frontend/src/components/Rules/index.tsx:35]: It is not!

**Comment by wittejm** [comment]
@KentShikama LMK if this looks good to go.

**Pull request merged**